src/factories/routePathLinkFactory.ts

```diff
@@ -6,8 +6,22 @@ export interface IRoutePathLinkResult {
     nodes: INode[];
 }
 
+interface IExternalRoutePathLinkNode {
+    linkkiByLnkverkkoAndLnkalkusolmuAndLnkloppusolmu: {
+        geojson: string,
+    };
+    lnkalkusolmu: string;
+    lnkloppusolmu: string;
+    lnkverkko: string;
+    relid: string;
+    reljarjnro: number;
+    solmuByLnkalkusolmu: Object;
```

Comment:
Again rather than Object, shouldn't these have a proper type?

Reply:
Outdated, it is IExternalNode now